term.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/pkg/term/termios"
 )
 
+const (
+	NONE     = iota // flow control off
```

Comment:
If you move the comments above the declaration, they'll show up in godoc

// Flow control is disabled
NONE = iota

// Software flow control
XONXOFF